wandb/old/summary.py

```diff
@@ -31,7 +31,12 @@ def __init__(self, root=None, path=()):
             self._root = root
             json_dict = root._json_dict
             for k in path:
-                json_dict = json_dict[k]
+                if k in json_dict:
+                    json_dict = json_dict[k]
+                else:
+                    json_dict[k] = {}
+                    json_dict = json_dict[k]
```

Comment:
I'd simply do this?
```suggestion
                json_dict = json_dict.get(k, {})
```

Reply:
Regarding the deeply nested, [here is](https://wandb.ai/acyrtest/lotsofLogs/runs/yp4h9bzq/overview?workspace=user-artsiom)a quick test: